java/requirements.yaml

```diff
@@ -3,16 +3,6 @@ dependencies:
      version: ~6.3.5
      repository: '@stable'
      condition: postgresql.enabled
-   - name: job
```

Comment:
Can't we let it be a job and add hooks, i mean make annotations configurable in chart-job and configure them here .

Reply:
What would be the purpose? Maybe it's better to have a chat ...